src/shelly_hap_rgb.hpp

```diff
@@ -75,7 +89,12 @@ class RGBWLight : public Component, public mgos::hap::Service {
   std::vector<mgos::hap::Characteristic *> state_notify_chars_;
 
   mgos::Timer auto_off_timer_;
-  bool dirty_ = false;
```

Comment:
why are you removing this? this makes every SaveState call hit flash, even if there are no changes to the config.

Reply:
yes, SaveState is invoked periodically from the main status timer, it's only expected to actually save state when there are changes.